common/artifacts/000_wc_artifacts.txt

```diff
@@ -812,11 +1190,109 @@ chalice_of_trollbanes = {
 
 	health = 1
 
-	monthly_character_prestige = 1.5
+	monthly_character_prestige = 1
 
 	light_group_opinion = 5
 
 	picture = "GFX_inv_misc_trophy_argent"
+	slot = relic
+}
+orb_of_corruption = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+	intrigue = 2
+
+	combat_rating = 10
+
+	monthly_character_prestige = 0.5
+	monthly_character_piety = 0.25
+
+	general_opinion = -10
+	fel_group_opinion = 20
+	death_god_opinion = 20
+
+	picture = "GFX_orb_of_corruption"
+	slot = relic
+}
+spellbook_of_drekthar = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes			# maybe reduce requirements for shaman education?
+	
+		is_magic_class_trigger = yes	
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 2
+	stewardship = 2
+	learning = 5
+	fertility = 0.1
+	health = 0.1
```

Comment:
9 attributes, fertility and health!? It's OP! Don't know vanilla artifacts that give so much.

Reply:
Well, I added the little bit of fertility and health to bring it closer to cost of 24 we set for tier 4 artifacts, and it's still below... chalice of trollbanes is more expensive then this...

Reply:
reduced attributes to +1/+1/+4, added +15 combat rating and added learning 12 as requirement. prestige boosted and piety gain deleted